jumpscale/packages/vdc_dashboard/bottle/deployments.py

```diff
@@ -410,4 +410,12 @@ def redir(solution):
     return redirect(f"/vdc_dashboard/#{solution}")
 
 
+@app.route("/api/vdc/status", method="GET")
+@login_required
+def is_running():
+    return HTTPResponse(
+        j.data.serializers.json.dumps({"running": True}), status=200, headers={"Content-Type": "application/json"}
```

Comment:
will be always running ? shouldn't we check for the process or ports for example ?